docs/getting-started/user-guides/decrediton-setup.md

```diff
@@ -0,0 +1,54 @@
+# **Decrediton Setup Guide**
+
+Last updated for v0.8.2.
+
+This guide is intended to help you setup the `Decrediton` application. 
+
+**Prerequisites:**
+
+- Use the [Decrediton Install Guide](/getting-started/install-guide.md#decrediton) to install `Decrediton`.
+
+---
+
+`Decrediton` is a graphical user interface for `dcrwallet`. When this application launches, it automatically starts its own instance of `dcrd` and `dcrwallet` in the background - it will not open if there is already a running instance of `dcrd`. Please note that as of 0.8.2, this application is a beta release - there are many bugs. 
```

Comment:
In the previous place you said 'there are know bugs' while here you say there are many bugs.  Might want to sick with one way (I prefer known bugs to many bugs personally).